libraries/botbuilder-ai/src/luisRecognizer.ts

```diff
@@ -265,13 +265,30 @@ export class LuisRecognizer implements LuisRecognizerTelemetryClient {
 
     /**
      * Creates a new LuisRecognizer instance.
-     * @param application An object conforming to the [LuisApplication](#luisapplication) definition or a string representing a LUIS application endpoint, usually retrieved from https://luis.ai.
-     * @param options (Optional) options object used to control predictions. Should conform to the [LuisRecognizerOptions](#luisrecognizeroptions) definition.
-     * @param includeApiResults (Deprecated) flag that if set to `true` will force the inclusion of LUIS Api call in results returned by [recognize()](#recognize). Defaults to a value of `false`.
+     * @param application The LUIS application endpoint, usually retrieved from https://luis.ai.
+     * @param options (Optional) Options object used to control predictions. Should conform to the [LuisRecognizerOptions](#luisrecognizeroptions) definition.
+     * @param includeApiResults (Deprecated) Flag that if set to `true` will force the inclusion of LUIS Api call in results returned by [recognize()](#recognize). Defaults to a value of `false`.
```

Comment:
When testing this kind of reference in vs code I get an error. I don't know if this is the proper way to link to reference other classes.
![image](https://user-images.githubusercontent.com/38112957/94469154-fb250c80-019b-11eb-8e4e-71a70cd2ce0d.png)

Reply:
Yea, maybe I could remove the link reference, I replicated it from the previous comments

Reply:
Resolved